src/components/NewWorkspaceModal.ts

```diff
@@ -312,16 +319,23 @@ const NewWorkspaceModal = withDisplayName(
     };
 
     const isBillingProjectApplicable = (project: BillingProject): boolean => {
-      // Only support cloning a workspace to the same cloud environment. If this changes, also update
-      // the Events.workspaceClone event data.
       // As of AJ-1164, if requireEnhancedBucketLogging is true, then azure billing projects are ineligible.
       // This coupling of enhanced bucket logging and billing project may change in the future.
-      return Utils.cond(
-        [!!workflowImport || !!requireEnhancedBucketLogging, () => !isAzureBillingProject(project)],
-        [!!cloneWorkspace && isAzureWorkspace(cloneWorkspace), () => isAzureBillingProject(project)],
-        [!!cloneWorkspace && isGoogleWorkspace(cloneWorkspace), () => isGoogleBillingProject(project)],
-        [Utils.DEFAULT, () => true]
-      );
+      if (!!workflowImport || !!requireEnhancedBucketLogging) {
+        return !isAzureBillingProject(project);
+      }
+      // Only support cloning a workspace to the same cloud environment. If this changes, also update
```

Comment:
nit: s/environment/platform/ to remain consistent with our nomenclature elsewhere.